src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--start-day-sun calendar--month-length-29">
+      <div class="calendar__day calendar__day--first"></div>
```

Comment:
the `calendar__day--first` class can be removed
```suggestion
      <div class="calendar__day"></div>
```

Reply:
![image](https://user-images.githubusercontent.com/90213755/227537025-71606d80-6da0-4751-a889-bb456082f911.png)
it can't, because of this class my margin-left for first element working